packages/v4/patternfly-docs/content/design-guidelines/components/tree-view/tree-view.md

```diff
@@ -27,19 +27,19 @@ A tree view can be used for:
 
 <img src="./img/treeview-usage.png" alt="Default tree view usage"  width="1004" />
 
-Use a tree view when:   
+### When to use
 * Selecting is the main use-case. 
 * You apply filters that need to be structured in a clear hierarchy. 
 * The data is structured into levels.
 * You need to display hierarchies that have more than 2 levels.
 
-Don’t use a tree view when: 
+### When not to use
 * You select 1 item from a small set. Instead, use [simple list](/components/simple-list), [select lists](/components/select), [radio buttons](/components/radio), [checkboxes](/components/checkbox), or a [dropdown](/components/dropdown).
 * You need to see an overview of a large amount of data.
 * You need to compare 2 large sets of data. Use [charts](/charts/about) for a better view. 
 * The hierarchy is too deep and includes hundreds of items. Instead, use a [drill-down menu](/components/menu#drilldown).
 
-## Tree view capabilities
+### Tree view capabilities
```

Comment:
I think this could arguably be renamed "Variations" and made into an H2?

Reply:
But then make sure the "Tree view in a page" "Tree view in a wizard" and "Tree view in a primary-detail" still stay under the "Usage" section - so basically shift them up